src/book/models/InvestmentAccount.ts

```diff
@@ -111,30 +108,39 @@ export default class InvestmentAccount {
     );
   }
 
-  get profitAbs(): Money {
+  get unrealizedProfitAbs(): Money {
     return this.value.subtract(this.cost);
   }
 
-  get profitAbsInCurrency(): Money {
+  get unrealizedProfitAbsInCurrency(): Money {
     return this.valueInCurrency.subtract(this.costInCurrency);
   }
 
-  get profitPct(): number {
+  get unrealizedProfitPct(): number {
     const n = Math.round(
-      (((this.profitAbs.toNumber() / this.cost.toNumber()) * 100) + Number.EPSILON) * 100,
+      (((this.unrealizedProfitAbs.toNumber() / this.cost.toNumber()) * 100) + Number.EPSILON) * 100,
     ) / 100;
     return n;
   }
 
-  get profitPctInCurrency(): number {
+  get unrealizedProfitPctInCurrency(): number {
     const n = Math.round(
       ((
-        (this.profitAbsInCurrency.toNumber() / this.costInCurrency.toNumber()) * 100
+        (this.unrealizedProfitAbsInCurrency.toNumber() / this.costInCurrency.toNumber()) * 100
       ) + Number.EPSILON) * 100,
     ) / 100;
     return n;
   }
 
+  get realizedProfitPct(): number {
+    const n = Math.round(
+      (
+        ((this.realizedProfit.toNumber() / this.totalBought.toNumber()) * 100) + Number.EPSILON
```

Comment:
This is a bit inaccurate I guess for positions that you close and re-open as it will count total bought for the stocks you haven't sold yet. Regardless, this is not shown to the user as then we would be showing unrealized profit (given the position is open).